src/pybind/mgr/dashboard/frontend/src/app/shared/components/wizard/wizard.component.ts

```diff
@@ -22,6 +22,16 @@ export class WizardComponent implements OnInit, OnDestroy {
   constructor(private stepsService: WizardStepsService) {}
 
   ngOnInit(): void {
+    this.initializeSteps();
+  }
+
+  ngOnChanges(changes: SimpleChanges): void {
+    if (changes.stepsTitle && !changes.stepsTitle.isFirstChange()) {
+      this.initializeSteps();
+    }
+  }
```

Comment:
Why do we have to to this? This is not an OnPush component, so the change detector should deal with it (unless we're doing tricky things).

Reply:
In the rgw mutilsite-wizard, there's a case where we dynamically determine the steps of the wizard based on whether the multi-cluster is configured or not, had to make this change to get it working.